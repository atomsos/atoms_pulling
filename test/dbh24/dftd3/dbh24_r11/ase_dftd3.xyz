3

O       0.543429315282048      0.552725897647972      0.750068539543519
H       0.567261638304295      0.618670641337658      4.662065655979356
H       0.008335798630000      0.015531576765635      4.253555250623787
