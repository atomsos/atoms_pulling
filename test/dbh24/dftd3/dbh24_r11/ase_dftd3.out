  _________________________________
                                   
 |         DFTD3 V3.1 Rev 1        |
 | S.Grimme, University Bonn       |
 |         October  2015           |
 |   see dftd3 -h for options      |
  _________________________________

 Please cite DFT-D3 work done with this code as:
 S. Grimme, J. Antony, S. Ehrlich and H. Krieg,
 J. Chem. Phys. 132 (2010), 154104
 If used with BJ-damping cite also
 S. Grimme, S. Ehrlich and L. Goerigk,
 J. Comput. Chem. 32 (2011), 1456-1465
 For DFT-D2 the reference is
 S. Grimme, J. Comput. Chem., 27 (2006), 1787-1799
 For DFT-D3M or DFT-D3M(BJ) the reference is
 D.G.A. Smith, L.A. Burns, K. Patkowski, and 
 C.D. Sherrill, J. Phys. Chem. Lett. 7 (2016) 2197-2203

  files read :     
 ase_dftd3.xyz
C6 coefficients used:
           2  C6 for element            1
Z=  1 CN= 0.912     C6(AA)=    3.03
Z=  1 CN= 0.000     C6(AA)=    7.59
           3  C6 for element            8
Z=  8 CN= 0.000     C6(AA)=   15.51
Z=  8 CN= 0.993     C6(AA)=   12.82
Z=  8 CN= 1.989     C6(AA)=   10.37

#               XYZ [au]               R0(AA) [Ang.]  CN       C6(AA)     C8(AA)   C10(AA) [au] 
   1   1.02693   1.04450   1.41742   o     1.241    0.000        15.5       311.8      7708.5
   2   1.07197   1.16912   8.81003   h     1.091    0.244         6.6        80.1      1185.6
   3   0.01575   0.02935   8.03805   h     1.091    0.244         6.6        80.1      1185.6

molecular C6(AA) [au] =        80.34

           DFT-D V3
 DF pbe                                               
 parameters
 s6       :    1.0000
 s8       :    0.7220
 rs6      :    1.2170
 rs18     :    1.0000
 alpha6   :   14.0000
 alpha8   :   16.0000
 k1-k3    :   16.0000    1.3333   -4.0000
 Cutoff   :   95.0000 a.u.
 CN-Cutoff:   40.0000 a.u.

 Edisp /kcal,au:    -0.1163 -0.00018531

 E6    /kcal :    -0.0933
 E8    /kcal :    -0.0230
 % E8        : 19.75

 doing analytical gradient O(N^2) ...

 |G|=   5.8454295254593890E-004
gdisp time     0.0
 Cartesian gradient written to file dftd3_gradient
 normal termination of dftd3
